draft-ietf-ppm-dap.md

```diff
@@ -524,7 +524,8 @@ else about the measurements. We call `F` the "aggregation function".
 This protocol is extensible and allows for the addition of new cryptographic
 schemes that implement the VDAF interface specified in
 {{!VDAF=I-D.draft-irtf-cfrg-vdaf-12}}. This protocol only supports VDAFs which
-require a single collection to provide useful results.
+require a single collection to provide useful results, and whose aggregation
+results are independent of the order in which measurements are aggregated.
```

Comment:
We should point to the relevant section of the VDAF draft. However we'll have to wait until VDAF-13 is out. Let's add a TODO to remind us, or file an issue?